.buildkite/longruns/pipeline.yml

```diff
@@ -253,8 +253,8 @@ steps:
         soft_fail: true
 
       - label: "MPI AMIP FINE: new target amip: topo + diagedmf"
-        key: "amip_target_topo_diagedmf_cpu"
-        command: "srun julia --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/run_amip.jl --config_file $CONFIG_PATH/amip_target_topo_diagedmf_cpu.yml --job_id amip_target_topo_diagedmf_cpu"
+        key: "amip_target_topo_diagedmf"
+        command: "srun julia --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/run_amip.jl --config_file $CONFIG_PATH/amip_target_topo_diagedmf.yml --job_id amip_target_topo_diagedmf_cpu"
```

Comment:
This config doesn't exist. Maybe you want to rename `amip_target_topo_diagedmf_cpu.yml` -> `amip_target_topo_diagedmf.yml`